dora/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -743,6 +746,39 @@ public void setOwner(String path, String user, String group) throws IOException
     }
   }
 
+  @Override
+  public void setAttribute(String path, String name, byte[] value) throws IOException {
+    if (StringUtils.isEmpty(name)) {
+      return;
```

Comment:
I guess `name` is the inode name like `file.txt`? So when someone trys to set with name "" we can print a warning or something?